lib/wcgi-runner/tests/integration.rs

```diff
@@ -0,0 +1,95 @@
+use std::path::Path;
+
+use bytes::Bytes;
+use http::{Request, StatusCode};
+use hyper::{body::HttpBody, Body};
+use tempfile::TempDir;
+use wcgi_host::CgiDialect;
+use wcgi_runner::Builder;
+
+const FERRIS_SAYS: &str = "https://registry-cdn.wapm.dev/packages/wasmer-examples/ferris-says/ferris-says-0.2.0-2f5dfb76-77a9-11ed-a646-d2c429a5b858.webc";
```

Comment:
We are guilty of this in Deploy as well, but we should stop depending on packages in a registry for tests - this will lead to pain down the line.

Doesn't necessarily need to be fixed for this PR, but we should strive to have the tests not depend on hosted packages in an external registry, but rather just have a test package / wasm in the repo.

Note that we are also working on a mock Rust registry that will be able to serve locally configured webcs like the real registry to test loaders.

Reply:
I think it's the least-bad solution here. I've been in projects where they commit binaries to the repository and it makes using the repository a massive pain. Recompiling test binaries all the time will also slow CI down a lot.